.vscode/settings.json

```diff
@@ -18,5 +18,14 @@
   "prettier.disableLanguages": ["javascript", "typescript"],
   "prettier.enable": true,
   "typescript.autoClosingTags": false,
-  "standard.enable": false
+  "standard.enable": false,
+  "cSpell.words": [
+    "camelcase",
+    "creds",
+    "kubernetes",
+    "openid",
+    "otomi",
+    "redkubes",
+    "robotv"
+  ]
```

Comment:
Ok, so you started to use cspell. I avoided this so far as we don't have user facing strings. But some output in the scripts can be seen as such, so adding cspell is fine. But can those be added to a `.cspell.json` config in the root of the project? Just like in the other public facing repos? And then also include the stuff that goes with it from `package.json`.

Reply:
done